### ISSUE TITLE
Chart shows non-existent value? log_metric don't match keras output in experiment.

### ISSUE THREAD

**Issue by bw4sz** [comet-ml/issue-tracking#119]
Experiment is here: https://www.comet.ml/bw4sz/deepforest-retinanet/0103303cf4894547928d9f1feb2c456e

Chart has a large dip in loss that I wanted to investigate

<img width="1177" alt="screen shot 2018-10-03 at 3 06 20 pm" src="https://user-images.githubusercontent.com/1208492/46442309-103a4280-c71e-11e8-876c-60b81c7bd509.png">

But when looking at the logs, no such values exist? No loss values have 5 or 6 in the first digit. In fact the entire logs look disjunct from the graph?

<img width="1225" alt="1ddcab3a914f13710dda4bf738108fed _screen shot 2018-10-03 at 3 09 31 pm" src="https://user-images.githubusercontent.com/1208492/46442484-a53d3b80-c71e-11e8-9c5f-95386561ef97.png">

The experiment is still running, with the graphs plotting values which don't seem to exist in the keras logs.

**Comment by gidim** [comment]
@bw4sz thanks for reporting. looking into this now

**Comment by bw4sz** [comment]
I'll look into this more tomorrow, but I don't think it is a systemic issue with my code. A different run that is on-going doesn't show this behavior.

https://www.comet.ml/bw4sz/deepforest-retinanet/bb7810a44bb64c36993f509c34571f95/chart

It does have different hyperparameters (batch size, # of convolutional layers, etc), so perhaps this is related to memory? I'll have a deeper look when I get a sec.

**Comment by bw4sz** [comment]
keras reports average loss 

https://github.com/keras-team/keras/blob/5b6243485acc20cc36f2db4f258512c332d691ec/keras/utils/generic_utils.py#L407

so if there are rare bad batches of data, comet will detect them, but the logs may smooth over them. 

Non-comet issue.
